packages/sync/src/types.ts

```diff
@@ -1,11 +1,28 @@
+/**
+ * External dependencies
+ */
+import type * as Y from 'yjs';
+
+declare global {
+	interface Window {
+		__experimentalCollaborativeEditingSecret?: string;
+		wp: {
+			ajax: {
+				settings: {
+					url: string;
+				};
+			};
+		};
```

Comment:
Shouldn't these be optional? Otherwise TS will allow consumers of the type to directly access `wp.ajax....` without optional chaining.